core/object/script_language.cpp

```diff
@@ -496,6 +496,21 @@ String ScriptServer::get_global_class_cache_file_path() {
 	return ProjectSettings::get_singleton()->get_global_class_list_path();
 }
 
+String ScriptServer::get_current_script_backtrace() {
+	String trace;
+	for (int i = 0; i < get_language_count(); i++) {
+		ScriptLanguage *sl = get_language(i);
+		Vector<ScriptLanguage::StackInfo> stack = sl->debug_get_current_stack_info();
+		if (stack.size()) {
+			trace += "stack_language: " + sl->get_name();
+			for (int i = 0; i < stack.size(); i++) {
+				trace += "\n" + itos(i) + ": " + stack[i].func + " (" + stack[i].file + " : " + itos(stack[i].line) + ")";
```

Comment:
Seeing as how the existing error/warning printing formats file-with-line as `%s:%i` (as seen in your own screenshot), rather than `%s : %i`, the spaces in `" : "` should probably be removed.
```suggestion
				trace += "\n" + itos(i) + ": " + stack[i].func + " (" + stack[i].file + ":" + itos(stack[i].line) + ")";
```

Reply:
makes sense